.github/workflows/jekyll-gh-pages.yml

```
@@ -0,0 +1,51 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
```

might be better to update the example name to something like this
```suggestion
name: Deploy GitHub Pages
```